cattle/plugins/docker/storage.py

```diff
@@ -112,7 +111,4 @@ def _do_volume_remove(self, volume, storage_pool, progress):
                     shutil.rmtree(path)
 
     def _path_to_volume(self, volume):
-        host_path = volume.uri.replace('file://', '')
-        mounted_path = re.sub('^.*?/var/lib/docker', '/host/var/lib/docker',
-                              host_path)
-        return mounted_path
+        return volume.uri.replace('file://', '')
```

Comment:
Is there another change outside of python-agent that will make this work for boot2docker? Are we going to mount all of /mnt into the rancher-agent container?

To remind you of why this is like this, take a look at this snippet of a rancher api volume:
https://gist.github.com/cjellick/add572bf72b8dfe09475
The URI field starts with /mnt/sda1, which does not (currently) resolve from inside the rancher-agent container.